app/command.go

```diff
@@ -434,8 +434,8 @@ func (a *App) HandleCommandResponsePost(command *model.Command, args *model.Comm
 		post.AddProp("from_webhook", "true")
 	}
 
-	// Do not process text if this is a code block
-	skipSlackParsing := command.Trigger == "code"
+	// Do not process text if the command is built in or the response contains "skip_slack_parsing": true.
+	skipSlackParsing := builtIn || response.SkipSlackParsing
```

Comment:
The only build in command that need this is [`code`](https://github.com/mattermost/mattermost-server/blob/master/app/command_code.go)